SPECS/reaper/reaper_build_caches.sh

```diff
@@ -0,0 +1,142 @@
+#! /bin/bash
+
+set -eux pipefail
+
+# NOTE: EXECUTING with id=0 (root) CAUSES REAPER BUILD TO FAIL.
+# Cassandra reaper version for which to generate build caches.
+# Set this to the required version.
+VERSION="3.1.1"
+# URL to download the sources to build
+SOURCE_URL="https://github.com/thelastpickle/cassandra-reaper/archive/refs/tags/${VERSION}.tar.gz"
+
+# Build cache names
+BOWER_COMPONENTS="reaper-bower-components-${VERSION}.tar.gz"
+SRC_UI_NODE_MODULES="reaper-srcui-node-modules-${VERSION}.tar.gz"
+BOWER_CACHE="reaper-bower-cache-${VERSION}.tar.gz"
+MAVEN_CACHE="reaper-m2-cache-${VERSION}.tar.gz"
+NPM_CACHE="reaper-npm-cache-${VERSION}.tar.gz"
+LOCAL_LIB_NODE_MODULES="reaper-local-lib-node-modules-${VERSION}.tar.gz"
+LOCAL_N="reaper-local-n-${VERSION}.tar.gz"
+
+tempDir=$(mktemp -d)
```

Comment:
Please also do a clean-up. It's a temporary directory, but there's no reason to leave it behind:
```suggestion
tempDir=$(mktemp -d)
function clean_up {
    echo "Cleaning up..."
    rm -rf "$tempDir"
}
trap clean_up EXIT SIGINT SIGTERM
```

Reply:
Added.